addon/styles/addon.css

```diff
@@ -63,6 +63,10 @@
           flex: 1 1 auto;
 }
 
+.lt-infinity {
+  min-height: 1px;
+}
```

Comment:
Do we still need this?

Reply:
I think one of the issues is in a scrollable container, there is no height for the IntersectionObserver to get on screen.  I *think*.  Will keep playing around.

Reply:
I would maybe make this the height of the icon that indicates we are loading more rows.